### ISSUE TITLE
[frontend]Bootstrapping the UI work for cms springboot app

### ISSUE THREAD

**Issue by Sanskar95** [sastix/cms#11]
I feel react would be the best choice, some question regarding that:
1) Is there any mockup available?
2)Would it be a totally different container?
3)If mock up is not available would a generic dashboard with some specific features to the application work?

**Comment by iskitsas** [comment]
Hi @Sanskar95,

We do not have any mockup yet, but there are ready to use templates or best practices. Based on the framework you choose (eg React) you should follow the proposed controls. The frontend will have to support basic CRUD functionalities and be secure.

So, yes you can use a nice generic dashboard, it is up to you.

The host of this Frontend should be a different container, we would like not to mix it with the java backend, and have a clean independent build process for it.

**Comment by Sanskar95** [comment]
@iskitsas yeah sure creating a separate deployment for frontend sounds good , all the CRUD operations , role based login can be easily implemented in the app.
This design looks pretty smooth 
https://demos.creative-tim.com/material-dashboard-react/?_ga=2.109405569.586406127.1616888113-1699679312.1616888113#/admin/dashboard
Its responsive and even can be viewed as a progressive web app if there such a need.

**Comment by iskitsas** [comment]
linked to #20

**Comment by Sanskar95** [comment]
![react-router](https://user-images.githubusercontent.com/8667334/113279449-5f5b9100-92e3-11eb-9ffe-6e757340a6cd.png)

This is a very high level flow of webapp
